Buildings/Resources/Scripts/Dymola/ConvertBuildings_from_6_to_7.0.0.mos

```diff
@@ -80,6 +80,14 @@ convertElement(
     "Buildings.ThermalZones.ReducedOrder.EquivalentAirTemperature.VDI6007WithWindow",
     "alphaWinOut",
     "hConWinOut");
+convertElement(
+    "Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Reheat.SystemRequests",
+    "uDam",
+    "yDam_actual");
+convertElement(
+    "Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller",
+    "uDam",
+    "yDam_actual");
```

Comment:
This non-backward compatible change must also be documented in the release notes of the top-level `package.mo` file.

Reply:
resolved